deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/http/HttpResponse.java

```diff
@@ -16,10 +16,10 @@
  */
 public class HttpResponse
 {
-    protected final int status;
-    protected final byte[] body;
-    protected final byte[] errorReason;
-    protected final Map<String, String> headerFields;
+    private final int status;
```

Comment:
Curious if we never return this back to the user.

Reply:
We do in some cases, but the field itself can be private since we have a public getter